docs/api.md

```diff
@@ -489,9 +538,20 @@
 
 ## Schemas
 
+### User 
+```
+{
+  "id" : string
+  "userName" : string,
+  "email" : string,
+  "password" : string
+}
+```
+
 ### Event
 ```
 {
+  "userId": string,
```

Comment:
I think we don't need `userId` for every non-user schema.

Reply:
can remove for active template as it contains templateId, where else you think we can remove it?

Reply:
We can say that all routes related to events, templates, tags, stats are secured and scoped to the authenticated user.

Reply:
Okay..., have removed it from the schemas.